### PR TITLE
Added interface redundancy group to 1.6 release overview

### PR DESCRIPTION
**Pull request by HanlinMiao**
# Closes: #n/a
# What's Changed

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: nautobot/docs/release-notes/version-1.6.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ A new Custom Field type, "Markdown", has been added. Custom fields of this type
 
 APIs have been added to allow for caching of the results of looking up an object's content-type or Dynamic Group memberships, as well as for looking up the members of a Dynamic Group itself. These caches are disabled by default but can be enabled by configuring the [`DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT`](../configuration/optional-settings.md#dynamic_groups_member_cache_timeout) and [`CONTENT_TYPE_CACHE_TIMEOUT`](../configuration/optional-settings.md#content_type_cache_timeout) settings respectively. Apps (plugins) that make use of dynamic groups should review the [documentation for the APIs](../models/extras/dynamicgroup.md#membership-and-caching) to determine how and when to make use of the cache for improved performance.
 
+#### Interface Redundancy Group ([#4177](https://github.com/nautobot/nautobot/pull/4177))
+
+Interface Redundancy Group model and related views have been added to allow logical grouping of multiple interfaces under a specific interface redundancy protocol (HSRP, VRRP, CARP, and etc).
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.6.md:21]: I would request a link to the model documentation but it looks like we never added `docs/models/dcim/interfaceredundancygroup.md`... :-(

**Review by glennmatthews — APPROVED** [review]

File: nautobot/docs/release-notes/version-1.6.md (hunk 1 above)
Comment (HanlinMiao) [reply nautobot/docs/release-notes/version-1.6.md:21]: I will write a follow-up issue for this. 🫠

Comment (HanlinMiao) [reply nautobot/docs/release-notes/version-1.6.md:21]: #4201

File: nautobot/docs/release-notes/version-1.6.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ A new Custom Field type, "Markdown", has been added. Custom fields of this type
 
 APIs have been added to allow for caching of the results of looking up an object's content-type or Dynamic Group memberships, as well as for looking up the members of a Dynamic Group itself. These caches are disabled by default but can be enabled by configuring the [`DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT`](../configuration/optional-settings.md#dynamic_groups_member_cache_timeout) and [`CONTENT_TYPE_CACHE_TIMEOUT`](../configuration/optional-settings.md#content_type_cache_timeout) settings respectively. Apps (plugins) that make use of dynamic groups should review the [documentation for the APIs](../models/extras/dynamicgroup.md#membership-and-caching) to determine how and when to make use of the cache for improved performance.
 
+#### Interface Redundancy Group ([#4177](https://github.com/nautobot/nautobot/pull/4177))
```
Comment (bryanculver) [inline nautobot/docs/release-notes/version-1.6.md:19]: Link to the issue please: https://github.com/nautobot/nautobot/issues/2825

**Review by bryanculver — CHANGES_REQUESTED** [review]

File: nautobot/docs/release-notes/version-1.6.md (hunk 2 above)
Comment (bryanculver) [inline nautobot/docs/release-notes/version-1.6.md:19]:
```suggestion
#### Interface Redundancy Group ([#2825](https://github.com/nautobot/nautobot/issues/2825))
```

**Review by bryanculver — APPROVED** [review]

**Review by bryanculver — CHANGES_REQUESTED** [review]
Going to add @whitej6 to the commit of this so he gets credit.

**Review by bryanculver — APPROVED** [review]

**Pull request merged**